runt.toml

```diff
@@ -390,6 +388,32 @@ fud e --from systolic --to jq \
 """
 expect_dir = "tests/correctness/systolic/mmult-expect"
 
+[[tests]]
+name = "[correctness] `@fast` attribute should not change program behavior"
+paths = ["tests/correctness/fast-attr/valid/*.futil"]
+cmd = """
+fud exec --from calyx --to jq \
+         --through dat \
+         --through icarus-verilog \
+         -s calyx.flags "-p validate -p compile -p lower" \
```

Comment:
Just a drive-by comment: it might be worth disabling `static-promotion` on these tests; a promoted `static` seq will follow the `@fast` invariant, but `static` seqs get compiled before `tdcc`. Disabling `static-promotion` will make sure the tdcc changes you made are being tested.